uniffi/src/lib.rs

```diff
@@ -354,6 +358,76 @@ unsafe impl ViaFfi for String {
     }
 }
 
+/// Support for passing timestamp values via the FFI.
+///
+/// Timestamps values are currently always passed by serializing to a buffer.
```

Comment:
Could you please add a brief comment here about the use of an `i64` to represent times before the unix epoch? It's a detail that's otherwise easy to miss for casual readers.

Reply:
I gave it a shot :sweat_smile: